lua/entities/starfall_processor/init.lua

```diff
@@ -19,6 +19,15 @@ function ENT:UpdateTransmitState()
 	return TRANSMIT_ALWAYS
 end
 
+function ENT:ValidateCustomModel(model)
```

Comment:
Can you make this a global function, SF.CanModel in sflib.lua that takes parameters: player, model. Also change IsValid(self.owner) to player:IsValid() and remove IsPlayer check?

Reply:
Also add `model = SF.NormalizePath(string.lower(model))` to the start of it

Reply:
`return util.IsValidModel(model) and util.IsValidProp(model) and player:IsValid() and hook.Run("PlayerSpawnObject", self.owner, model)~=false`